src/java/io/bazel/rulesscala/jdeps/JdepsWriter.java

```diff
@@ -0,0 +1,30 @@
+package io.bazel.rulesscala.jdeps;
+
+import com.google.devtools.build.lib.view.proto.Deps;
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class JdepsWriter {
```

Comment:
I think that according to whole ceremony this class should be `final` with private constructor. But I'm fine with the way it is now.